Let Marcos monitor various directories for changes via CODEOWNERS

marcoscaceres opened the pull request:
#### 72e50506cdd58880c70a3b4c05001fccfb8236ac
<pre>
Let Marcos monitor various directories for changes via CODEOWNERS
<a href="https://bugs.webkit.org/show_bug.cgi?id=246200">https://bugs.webkit.org/show_bug.cgi?id=246200</a>
rdar://100880896

Reviewed by Jonathan Bedard.

* .github/CODEOWNERS:
* metadata/contributors.json:

Canonical link: <a href="https://commits.webkit.org/257163@main">https://commits.webkit.org/257163@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/3032751349a799efd9e4afb09a44b48733427d08

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/6/builds/97222 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/77/builds/6489 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/43/builds/30365 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/8/builds/106741 "Built successfully") | 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/76/builds/6774 "Built successfully") | [✅ 🛠 mac-debug](https://ews-build.webkit.org/#/builders/71/builds/35222 "Built successfully") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/36/builds/89692 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/12/builds/103428 "Built successfully") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/19/builds/102888 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/78/builds/5091 "Passed tests") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/61/builds/83849 "Built successfully") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/35/builds/32138 "Passed tests") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/9/builds/86941 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/3/builds/88764 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/34/builds/75064 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/81/builds/510 "Built successfully") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/73/builds/20240 "Passed tests") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/82/builds/493 "Built successfully") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/70/builds/21691 "Passed tests") | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/74/builds/4932 "Built successfully and passed tests") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/80/builds/5293 "Built successfully") | [❌ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/60/builds/44187 "Found 1 new test failure: fast/images/animated-heics-draw.html (failure)") | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/79/builds/1734 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/62/builds/41012 "Passed tests") | | 
<!--EWS-Status-Bubble-End-->

JonWBedard commented on .github/CODEOWNERS:
```diff
@@ -96,8 +96,8 @@
-/Source/WebCore/Modules/paymentrequest @dcrousso
-/LayoutTests/http/tests/paymentrequest @dcrousso
+/Source/WebCore/Modules/paymentrequest @dcrousso @marcoscaceres
+/LayoutTests/http/tests/paymentrequest @dcrousso @marcoscaceres
```
Huh, this is a bit surprising.

According to https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners, code owners must have write access to the repository. I would expect it was triage access that's required, and I think we should actually file a GitHub bug about that

marcoscaceres replied:
@JonWBedard, I've been granted committer status, so... maybe that will grant me write access?

marcoscaceres committed:
Let Marcos monitor various directories for changes via CODEOWNERS
https://bugs.webkit.org/show_bug.cgi?id=246200
rdar://100880896

Reviewed by Jonathan Bedard.

* .github/CODEOWNERS:
* metadata/contributors.json:

Canonical link: https://commits.webkit.org/257163@main
